package.json

```diff
@@ -54,6 +54,7 @@
   "scripts": {
     "prepublish": "gulp prepublish",
     "pretest": "gulp pretest",
+    "prestart": "webdriver-manager start > /dev/null 2>&1 &",
```

Comment:
It might be better if we created a child process of webdriver-manager and then stopped the process when the tests are completed.